src/test/java/com/matchus/domains/user/service/UserServiceTest.java

```diff
@@ -75,4 +82,91 @@ void checkNickname() {
 		assertThat(successResponse.isIsduplicated()).isFalse();
 	}
 
+	@Test
+	@DisplayName("유저 탈퇴 요청 테스트")
+	void deactivateUser() {
```

Comment:
```suggestion
	void deactivateUserTest() {
```
뒤에 Test 붙여주시면 컨벤션 지켜질 수 있을지도??

Reply:
앗...